pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -46,6 +46,7 @@
 , withBs2b ? withFullDeps # bs2b DSP library
 , withBzlib ? withHeadlessDeps
 , withCaca ? withFullDeps # Textual display (ASCII art)
+, withCdio ? withFullDeps && withGPL # Audio CD grabbing
```

This sounds like something we'd want in regular ffmpeg.

```suggestion
, withCdio ? withHeadlessDeps && withGPL # Audio CD grabbing
```

---

None of Arch, Fedora, or Debian enable it, and although `libcdio` still gets a reasonable volume of commits, it hasn’t seen a stable release in over half a decade. I think leaving it in the full variant is best for now.